tools/ci_build/github/azure-pipelines/templates/rocm.yml

```diff
@@ -0,0 +1,149 @@
+parameters:
```

Comment:
Contents of this file are copied from https://github.com/microsoft/onnxruntime/blob/master/tools/ci_build/github/azure-pipelines/orttraining-py-packaging-pipeline-rocm.yml